### DISCUSSION TITLE
How to build locally?

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-style-spec#363] [Q&A]
When I try to run `npm install` I get:

> TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/bart/src/maplibre-gl-style-spec/build/generate-style-spec.ts

I am using Node.js v20.6.1. Are there any special setup instructions?

**Comment by HarelM** [comment]
Try codesoaces, might be something locally on your machine.
CI passes, so you can look there, maybe it uses node 18.
